addons/sourcemod/scripting/szf/pickupweapons.sp

```diff
@@ -211,12 +211,13 @@ bool AttemptGrabItem(int iClient)
 		char sClient[128];
 		GetClientName2(iClient, sClient, sizeof(sClient));
 		
-		if (iIndex == 9 || iIndex == 10 || iIndex == 12)	//Shotgun
+		if (iIndex > 8 && iIndex < 13)	//Shotgun
```

Comment:
`9 <= iIndex <= 12` reads better for me as it between 9 to 12, dunno what others think about this